trams_api_spec/trams_api_spec.yaml

```diff
@@ -842,12 +842,12 @@ definitions:
         $ref: "#/definitions/Placeholder"
       concerns:
         $ref: "#/definitions/Placeholder"
-  AcademyListItem:
+  EstablishmentListItem:
     type: "object"
     properties:
       urn:
         type: "string"
-        description: "Unique reference for an Academy"
+        description: "Unique reference for an Establishment"
```

Comment:
I think we should add `UKPRN` to the summary as well - so it returns Name, URN and UKPRN. I think this will be more useful for consumers